about.html

```diff
@@ -0,0 +1,150 @@
+
+                  <img onclick="clickToSwitch()" alt="harmburger" src="images/harmburger_img.png" />
+
+                </div>
+                <div class="desktop">
+                  <ul class="nav-icons">
+                    <li><a href="#program">Program</a></li>
+                    <li><a href="./index.html">Home</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#sponsor">Sponsor</a></li>
+                    <li><a href="#news">News</a></li>
+                  </ul>
+                </div>
+              </div>
+
+              <!--Mobile blue menu-->
+            <div class="mobile-menu-container" id="switch-display">
+              <div class="mobile-items">
+                <div id="mobile-x-icon">
+                  <img onclick="clickToSwitch()" alt="close-icone-image" src="images/x-icon.png" />
+                </div>
+                <div class="mobile-menu">
+                  <ul class="mobile-list">
+                    <li><a onclick="clickToSwitch()" href="#program">Program</a></li>
+                    <li><a onclick="clickToSwitch()" href="./index.html">Home</a></li>
+                    <li><a onclick="clickToSwitch()" href="#join">Join</a></li>
+                    <li><a onclick="clickToSwitch()" href="#sponsor">Sponsor</a></li>
+                    <li><a onclick="clickToSwitch()" href="#news">News</a></li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+
+            </nav>
+          </div>
+        </header>
+        <section class="head-part" id="head">
+          <h1 class="mobile">
+            About Africa<br/>Tech
+            Week
+          </h1>
+          <h2 class="head-h2 desktop">
+            About Africa <br/>Tech
+            Week
+          </h2>
+          <p class="head-part-text">
+            The expert panels will cover public and private policy aspects and their implications advantages and opportunities. It will also include industry’s leaders from the private sector, civil society, investors, partners, experts, academics all with their expert insight into the different ways artificial intelligence,cryptocurrency,IoT, will affect the future of African Development.
+          </p>
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4" ><strong>25-29 OCTOBER 2022</strong></h4>
+          <h4 class="head-h4">Virtual Event</h4>
+        </section>
+      </div>
+
+      <section id="stuff" class="text-center">
+        <h4>Afri Tech Week Logo 2022</h4>
+        <p>The main sessions will feature addresses from world authorities on aspects of the 4th Industrial Revolution (Industry 4.0). They will outline the challenges and different scenarios as well as the mind blowing economic opportunities Industry 4.0 can deliver for those countries and organisations that make preparations for it.</p>
+        <img src="./images/afri-tech-img.png" alt="">
+      </section>
```

Comment:
- [ ] As required for the project and from the template design, kindly make sure, the **About Page** contains a top section with a **centered column** that includes **a page title**, **a text block with the page description**, and **a line with a contact information or any other prompt under it**.

| Template Design | Your Design |
| ---------------------- | -------------------- |
| ![Screenshot from 2022-09-17 00-31-39](https://user-images.githubusercontent.com/83042742/190826473-ff80a55b-4fe5-4a9a-93e3-6b72ba2c253b.png) | ![Screenshot from 2022-09-17 00-30-29](https://user-images.githubusercontent.com/83042742/190826493-82a79950-ce60-4b54-bd7b-f2bd4b403f36.png) |